spec/support/api/context.rb

```diff
@@ -26,7 +26,7 @@
 shared_context "provider api", provider: true do
   let(:master) { provider && master_account }
   let(:provider) { FactoryBot.create(:provider_account, self_domain: 'example.org') }
-  let(:provider_key) { provider.provider_key }
+  let(:provider_key) { provider.api_key? ? provider.provider_key : provider.reload.provider_key }
```

Comment:
Can we always reload?

Reply:
I do not know but I didn't try out of efficiency... should we always reload? why?

Reply:
I just thought about it for simplicity (of the code). Of course, it's another select to the db.